### PR TITLE
feat: add flypadabout command for flyPadOS and aircraft version

### PR DESCRIPTION
**Pull request by pdellaert**
## Description

A command to explain to users how to get a picture from the About page of the flyPad / EFB to help with troubleshooting.

## Test Results

Screen Shot 2022-10-01 at 1 45 22 PM
Screen Shot 2022-10-01 at 1 45 34 PM
Screen Shot 2022-10-01 at 1 45 46 PM
Screen Shot 2022-10-01 at 1 45 57 PM

## Discord Username

straks#7240

### CODE REVIEW

**Comment by benw202** [comment]
Looking good! Can you add the image to `/assets/images/support`

**Comment by pdellaert** [comment]
> Looking good! Can you add the image to `/assets/images/support`

Done :)

File: .github/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 Update <small>_ September 2022</small>
 
+- feat: add flypadabout command for flyPadOS and aircraft version (02/10/2022)
```
Comment (NathanInnes) [inline .github/CHANGELOG.md:5]: Check date :)

Comment (pdellaert) [reply .github/CHANGELOG.md:5]: Lol! Nice catch... 🤦 for me here :D

**Pull request merged**